portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -36,7 +36,7 @@
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        Query find = new Query("comment").addSort("timestamp", SortDirection.DESCENDING);
+        Query find = new Query("comment").addSort("date", SortDirection.DESCENDING);
         DatastoreService data = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Things like the DatastoreService and UserService can be stored as `private final` fields of the servlet object so you don't need to create an instance of them for every request.